### ISSUE TITLE
Please do not format my labels and categories

### ISSUE THREAD

**Issue by BrandonMathis** [react-ga/react-ga#57]
Hi! I noticed [the following lines of code in this repo](https://github.com/react-ga/react-ga/blob/master/src/index.js#L43-L63) are modifying the case and format of the text I pass to the `ga.event` method. I would really appreciate some explanation as to why this is helpful. Otherwise, I would recommend this functionality be removed from the codebase or configureable on/off.

Reason that I ask is because I find it quite frustrating that what I want to be sent to google analytics is not what is actually being sent. 

Specifically

```
ga.event({
  category: 'Log In Button',
  action: 'Button Click',
});
```
Is outputting "Log in Button" instead of "Log In Button" :(

I'm more than willing to submit a PR that removes this case change on 2 char words specifically but I would first like to learn if there is some deeper reason why this is being enforced.

**Comment by yangshun** [comment]
I think it's a Mozilla convention and I very much agree that this should be opt-in. Duplicate of this issue: https://github.com/react-ga/react-ga/issues/24. PR here: https://github.com/react-ga/react-ga/pull/34

**Comment by adamlofting** [comment]
This package started out as something for teams at Mozilla to use, and that decision was to help keep our event labels consistent.

I'm happy to change this now that the package is being used more widely.

**Comment by BrandonMathis** [comment]
Excellent! I would consider this closed once #34 is merged :)
